app/Models/User.php

```diff
@@ -78,6 +78,7 @@ class User extends Authenticatable implements CipherSweetEncrypted, FilamentUser
         'oriented_at',
         'suspended_at',
         'dismissed_customize_prompt_at',
+        'dismissed_browse_engagements_prompt_at',
```

Comment:
As we're collecting more of these, I'm wondering if they make sense to be in one of those schemaless attributes. Any thoughts on that?

Reply:
For now I was just thinking of the ones that keep track of the states of prompts. 

- `dismissed_customize_prompt_at`
- `dismissed_browse_engagements_prompt_at`
- `dismissed_browse_organizations_prompt_at`

If it makes sense to also do that for other things like the accept states and oriented and approved ones, we could consider that too.

Reply:
@greatislander are you still planning to make these model changes in this PR or will that be saved for another one?

Reply:
In this PR.